dask/dataframe/io.py

```diff
@@ -247,6 +247,22 @@ def from_array(x, chunksize=50000):
     return DataFrame(dsk, name, columns, divisions)
 
 
+from_dataframe_names = ('from-dataframe-%d' % i for i in count(1))
+
+
+def from_dataframe(df, npartitions=10):
+    nrows = len(df)
+    chunksize = int(ceil(float(nrows) / npartitions))
```

Comment:
Perhaps add `from __future__ import division`, and skip the manual float casting?